index.js

```diff
@@ -0,0 +1,13 @@
+function calculateTotal(items) {
+  items.forEach(item => Calculate(item))
+
+  return items
+}
+
+function Calculate(item) {
+  item.total = parseFloat(((item.price - (item.price * item.discount)) * item.quantity).toFixed(2))
```

Comment:
`toFixed` works but the fact that it type casts to a string is a little funky. This way is slightly better.

```JS
parseInt(unformattedTotal * 100) / 100
```

Reply:
Ok.